<launch>
    <!-- <include file="$(find tabletop)/launch/tabletop.launch"/> -->
    <include file="$(find object_recognition_v2)/launch/object_recognition.launch"/>
    <node name="main_state" pkg="main_state" type="main.py" output="screen" args="ManipulationTask" />
</launch>